installer/src/panels/com/jboss/devstudio/core/installer/JREPathPanel.java

```diff
@@ -56,9 +56,9 @@
 	private static final String winTestFiles[];
 	private static final String linTestFiles[];
 	private static final String gnuVersion = "gij ";
-	// TODO support jre10/jdk10 when it comes out
+
 	// see also JREPathValidator.java
-	private static final int MIN_VERSION = 8;
+	private static final int MIN_VERSION = 11;
 	private static final int MAX_VERSION = 12;
```

Comment:
Just wondering is this is exclusive:
- if yes then it's fine as we only allow Java11
- if no I think we need to update to the latest (Java 15)

Reply:
current behavior is to inform user that this version is not tested:
![Capture d’écran de 2020-09-29 14-48-28](https://user-images.githubusercontent.com/600414/94560762-5ff76a00-0263-11eb-93a9-bb6d6d13ebd8.png)

Reply:
@jeffmaury : given the actual state, should we update to Java 14 ( as it's the latest the test are run against )? 
CC @jkopriva @odockal WDYT ?

Reply:
If it allows Java up to 15 with a warning then it's fine because on our side the only supported version is Java11 (latest LTS)

Reply:
@sbouchet @jeffmaury If only java that is supported is 11 then it is fine for others to show warning, imho.